locales/en.yml

```diff
@@ -104,7 +159,7 @@ share.title: Share
 share.copy: Copy Link
 footer.about: About us
 footer.support: Support
-footer.privacy: Privacy
+footer.privacy: PrivacyS
```

Comment:
why did you add `S` at the end of word?
`PrivacyS` ?

Reply:
yes, it is a typo